docs/working-with-cdk-for-terraform/remote-backend.md

```diff
@@ -102,13 +102,13 @@ cdktf synth
 Move Terraform state file into the output directory.
 
 ```bash
-mv terraform.tfstate cdktf.out
+mv terraform.hello-terraform.tfstate cdktf.out/stacks/hello-terraform
 ```
 
 Run Terraform init
```

Comment:
Should we mention cdktf diff and deploy here as well?

Reply:
Yeah we might want to revisit other docs as well, I kept this out of scope for this, as this PR was about fixing the wrong paths since the multiple stacks feature